src/pages/Shop.js

```diff
@@ -3,30 +3,28 @@ import { data } from "../data";
 
 const Shop = ({ addToCart, updateCart }) => {
   return (
-    <>
-      <div className="product-list">
-        {data.map((product) => (
-          <div key={product.id} style={{ margin: "20px" }}>
-            <img src={product.img} alt="shop-images" className="shop-img" />
-            <br />
-            <span style={{ fontWeight: "bold" }}>{product.name}</span>
-            <span style={{ float: "right", marginRight: "10px" }}>
-              $ {product.price}
-            </span>
-            <br />
-            <button
-              onClick={() => {
-                addToCart();
-                updateCart(product);
-              }}
-              className="button"
-            >
-              add to cart
-            </button>
-          </div>
-        ))}
-      </div>
-    </>
+    <div className="product-list">
+      {data.map((product) => (
+        <div key={product.id} style={{ margin: "20px" }}>
```

Comment:
the `margin: "20px"` here can be an integer. It doesn't have to be a string. Both work.